gulpfile.babel.js

```diff
@@ -0,0 +1,76 @@
+import gulp from 'gulp';
+import chug from 'gulp-chug';
+import yargs from 'yargs';
+
+const argv = yargs
+  .options({
+    rootPath: {
+      description: '<path> path to web assets directory',
+      type: 'string',
+      requiresArg: true,
+      required: false,
+    },
+    vendorPath: {
+      description: '<path> path to vendor directory',
+      type: 'string',
+      requiresArg: true,
+      required: false,
+    },
+    nodeModulesPath: {
+      description: '<path> path to node_modules directory',
+      type: 'string',
+      requiresArg: true,
+      required: false,
+    },
+  })
+  .argv
+;
+
+const config = [
+    '--rootPath',
+    argv.rootPath || '../../../../web/assets/',
+    ...(argv.vendorPath ? [
+        '--vendorPath',
+        argv.vendorPath + 'sylius/sylius/src/Sylius/Bundle/',
```

Comment:
I've added this to allow using a custom `vendorPath` from the parent gulpfile. WDYT @pamil?